BAZProject/MVVM/Controllers/HomeController/HomeView.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  HomeView.swift
+//  BAZProject
+//
+//  Created by victor manzanero on 07/11/22.
+//
+
+import UIKit
+import Bond
+
+class HomeViewController: UIViewController {
```

Comment:
Marca tus clases como final
```suggestion
final class HomeViewController: UIViewController {
```